client/jetpack-cloud/sections/partner-portal/payment-method-delete-dialog/index.tsx

```diff
@@ -21,11 +21,16 @@ const PaymentMethodDeleteDialog: FunctionComponent< Props > = ( {
 } ) => {
 	const translate = useTranslate();
 
-	const paymentMethodSummary = getPaymentMethodSummary( {
-		translate,
-		type: paymentMethod?.card.brand,
-		digits: paymentMethod?.card.last4,
-	} );
+	const paymentMethodSummary = (
+		<>
+			{
```

Comment:
Minor: you can remove the curly braces (and the Fragment tags) around this now that it's pure JSX.

Reply:
Fixed.